xonsh/__init__.py

```diff
@@ -1,5 +1,11 @@
 __version__ = '0.5.3'
 
+import sys as _sys
+if '--timings' in _sys.argv:
+    from xonsh.timings import setup_timings
+    setup_timings()
+del _sys
```

Comment:
Ahh. Yes. I was afraid it was so. 

Is the clock thing also true for Windows. Don't worry I will check my self ...